TheWarWithin/Priorities/RogueOutlaw.simc

```diff
@@ -4,127 +4,123 @@
 actions.precombat+=/apply_poison
 actions.precombat+=/use_item,name=imperfect_ascendancy_serum
 actions.precombat+=/stealth,precombat_seconds=2
-# Builds with Keep it Rolling prepull Adrenaline Rush before Roll the Bones to consume Loaded Dice immediately instead of on the next pandemic roll
-actions.precombat+=/adrenaline_rush,precombat_seconds=2,if=talent.improved_adrenaline_rush&buff.adrenaline_rush.down&talent.keep_it_rolling&talent.loaded_dice
-actions.precombat+=/roll_the_bones,precombat_seconds=2,if=buff.roll_the_bones.down
-actions.precombat+=/adrenaline_rush,precombat_seconds=1,if=talent.improved_adrenaline_rush&buff.adrenaline_rush.down
+# Builds with Keep it Rolling+Loaded Dice prepull Adrenaline Rush before Roll the Bones to consume Loaded Dice immediately instead of on the next pandemic roll.
```

Comment:
Are the AR buff checks not needed any longer?  The reason they were here is because you could have AR up when moving pack to pack (vs. starting fresh for a boss fight).  We don't want to suggest AR or rerolling when you already have AR and RtB active.